copis/core.py

```diff
@@ -299,7 +313,7 @@ def pause(self) -> bool:
         # might be calling it from the thread itself
         try:
             self.imaging_thread.join()
-        except RuntimeError as e:
+        except:
```

Comment:
Is RuntimeError too specific here?

Reply:
No, I was trying to delint. seems I traded one lint warning for another. let me just add it back. we can flesh it our and print the error message later.